Makefile

```diff
@@ -3,31 +3,23 @@ VERSION = 3.2.0
 
 .DEFAULT_GOAL := all
 
-# Locate testables:
-_TESTABLES = $(shell find src/ -name '*_test.go'|xargs -I{} dirname {}|sed 's/src\///g'|uniq|sort)
-_COMPLIABLE = $(shell find src/ -name '*.go' | xargs -I{} dirname {}|sed 's/src\///g'|uniq|sort)
-
-GO_TESTS = \
-	$(addsuffix .test,$(_TESTABLES))
-
-BUILDABLES = \
-	mixer.build
-
-include Makefile.gobuild
-
-# We want to add compliance for all built binaries
-_CHECK_COMPLIANCE = $(addsuffix .compliant,$(_COMPLIABLE))
-
 # Ensure our own code is compliant..
-compliant: $(_CHECK_COMPLIANCE)
-install: $(BINS)
-	test -d $(DESTDIR)/usr/bin || install -D -d -m 00755 $(DESTDIR)/usr/bin; \
-	install -m 00755 bin/* $(DESTDIR)/usr/bin/.
+compliant:
+	go fmt ./... && go vet ./...
+
+install:
+	test -d $(DESTDIR)/usr/bin || install -D -d -m 00755 $(DESTDIR)/usr/bin;
+	install -m 00755 $(GOPATH)/bin/mixer $(DESTDIR)/usr/bin/.
 	install -m 00755 pack-maker.sh $(DESTDIR)/usr/bin/mixer-pack-maker.sh
 	install -m 00755 superpack-maker.sh $(DESTDIR)/usr/bin/mixer-superpack-maker.sh
 	install -D -m 00644 yum.conf.in $(DESTDIR)/usr/share/defaults/mixer/yum.conf.in
 
 release:
 	git archive --format=tar.gz --verbose -o mixer-tools-$(VERSION).tar.gz HEAD --prefix=mixer-tools-$(VERSION)/
 
-all: compliant $(BUILDABLES)
+all: compliant
```

Comment:
Are we expecting developers to be running make, or just devops? If developers are then a few more targets might be helpful, e.g. one to run metalinter and one to build without go vet.

Reply:
I intend to add that but not as part of this commit. This is more for devops, not so much developers IMHO.